boards/imix/src/main.rs

```diff
@@ -111,6 +119,18 @@ unsafe fn set_pin_primary_functions() {
     PC[29].configure(None);     // D4          -- GPIO Pin
     PC[30].configure(None);     // D3          -- GPIO Pin
     PC[31].configure(None);     // D2          -- GPIO Pin
+
+    // Enable, and disable output for RF233 pins
+    // IRQ
```

Comment:
Not entirely sure. 

From first glance, those pins were specifically configured that way in the Firestorm and config alone doesn't set the same state.

I don't have a imix or firestorm so I can't test.